code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -85,37 +85,29 @@
 
 		if("dab")
 			if(buckled)
-				to_chat(src, "You cannot dab while buckled in")
+				to_chat(src, "You cannot dab while buckled in.")
+				return
+			if(handcuffed)
+				to_chat(src, "You cannot dab while handcuffed.")
```

Comment:
`to_chat(src, "You cannot dab in your current state.")` is more generic to the many circumstances one could be unable to dab.